src/plugins/maps_dashboards/public/pages/map_page/map.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useRef } from 'react';
+import { Map as Maplibre } from 'maplibre-gl';
+import { MaplibreLayersPanelControl } from '../../components/layer_control_panel';
+import '../../index.scss';
+
+export const Map = () => {
+  const mapContainer = useRef(null);
+
+  useEffect(() => {
+    const mapStyle =
+      'https://dldbnqfps17cd.cloudfront.net/styles/basic-preview/compressedstyle.json';
+
+    const initialState = {
```

Comment:
will this remain hardcoded in the future too? Let's move it to constants file in that case :)

Reply:
This is only used for development for now, we'll change to use the production vector tiles service then. Will add comment here